### ISSUE TITLE
pretty REST URLs also for invitation links

### ISSUE THREAD

**Issue by giovanisp** [fairdemocracy/vilfredo-core#43]
As of now the invitation link is like 

Click to participate: http://vilfredo.org/invitation?email=name@example.com&eit=0e0c654353be4f55bc6293c915610f67

But we should use the REST pretty URLs methods and have a URL like:

Click to participate: http://vilfredo.org/invitation/0e0c654353be4f55bc6293c915610f67

**Comment by pietrosperoni** [comment]
much prettier!

**Comment by giovanisp** [comment]
better yet Click to participate: https://vilfredo.org/i/0e0c654353be4f55bc6293c915610f67

https for security and avoid useless redirect and  /i/ for "invitation" .. shorter and cleaner.

**Comment by pietrosperoni** [comment]
No. If not we go back to giving the sensation that anyone can participate and leave.

**Comment by giovanisp** [comment]
? No what? I just shortened the url and suggested https so the server does not need to do a useless http to https redirect. In fact the config file should have the prefix used (http or https) that is then used everywhere.

**Comment by pietrosperoni** [comment]
Sorry, I meant ome thing is to "ask to be invited" (or "humbly beg to be admitted and participate" ;-) )
another is "click to participate". The more you make it look like an action they can do and stop doing, the more people will just not do it.
